src/Illuminate/Database/Connection.php

```diff
@@ -845,6 +846,17 @@ protected function getElapsedTime($start)
         return round((microtime(true) - $start) * 1000, 2);
     }
 
+    /**
+     * Get the affected row count.
+     *
+     * @param  mixed  $result
+     * @return int|null
+     */
+    protected function getRowCount($result)
```

Comment:
In variable `rowS`, in method name `row`. Not consistent naming.

Reply:
>Maybe it would be better to name it rowsCount? Or even affectedRowsCount?
>Then we do not need to write a comment, it will be much clear what this variable contains.

I noticed it, I was guided by the comment where you suggested that I should put it like that, I am fixing it along with other things

Reply:
I see that the correct thing is without the letter `S`
https://github.com/laravel/framework/blob/e80bc1ac5c4994e658b881f4c9d203571ada369a/src/Illuminate/Database/Connection.php#L608
`PDOStatement::rowCount` https://www.php.net/manual/en/pdostatement.rowcount.php